scripts/configure.sh

```diff
@@ -2,9 +2,10 @@
 
 HOME_DIR=$1
 OUTFILE=$2
+INFILE=$HOME_DIR/etc/qdrouterd.conf.in
 
-function swapVars() {
-  sed -i "s/\${HOSTNAME}/$HOSTNAME/g" $1
+function substVars() {
+    DOLLAR='$' envsubst < $1 > $2
```

Comment:
I'm ok with this, but if we defined the function as:

function substVars() {
    DOLLAR='$' envsubst < $1 > /tmp/qdrouterd.conf
    mv /tmp/qdrouterd.conf  $1
}

that would to my mind be slightly cleaner (but this is likely personal preference and influenced by history of the file).